src/Test/Decompile.hs

```diff
@@ -0,0 +1,124 @@
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Decompile where
+
+import Test.Tasty
+import Test.Tasty.ExpectedFailure
+import Test.Tasty.HUnit
+import EVM.Solidity
+import EVM.Solvers
+import Data.List.NonEmpty qualified as NE
+import Data.Map qualified as Map
+import Data.Maybe
+import Data.String.Here
+import Data.Text (Text)
+import Data.Text.IO qualified as T
+import Data.Validation
+
+import Act.Decompile
+import Act.Print
+import Act.CLI
+import Act.HEVM_utils
+
+import qualified EVM.Solvers as Solvers
+import EVM.Effects
+
+decompilerTests :: TestTree
+decompilerTests = testGroup "decompiler"
+  [ expectFail $ testCase "noop" $ checkDecompilation "C" [i|
```

Comment:
why is this one `expectFail`?

Reply:
IIRC because we do not have a way to represent empty behaviours in Act

Reply:
I think it's fine to leave that for another PR.